zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -113,7 +113,7 @@ pub trait WriteDisk {
         K: IntoDisk + Debug,
         V: IntoDisk;
 
-    /// Remove the given key from rocksdb column family if it exists.
+    /// Remove the given key form rocksdb column family if it exists.
```

Comment:
Shouldn't it be from?

```suggestion
    /// Remove the given key from rocksdb column family if it exists.
```